db/seeds.rb

```diff
@@ -63,7 +63,7 @@
                             contact: contact,
                             user: [user, nil].sample,
                             is_urgent: [true, false].sample,
-                            completed_on: [nil, [1,2,3].sample.days.ago].sample
+                            status: ['to_do', 'complete'].sample
```

Comment:
Shouldn't we use `Need.status.sample` instead so other statuses get included in the seeds?

Reply:
Thanks